### DISCUSSION TITLE
Problem connecting localhost network to Metamask

### DISCUSSION THREAD

**Discussion by DrBailey27** [Cyfrin/foundry-full-course-cu#933] [Q&A] [answered]
This is the error message I am getting when I try to connect my localhost network on my terminal on VSC
, , , 
"error sending request for url (http://127.0.0.1:8545/): error trying to connect: tcp connect error: Connection refused (os error 111)"
 , , ,
in addition to that 
Meta mask is not connecting to my localhost network that I created manually

can someone please help me solve this issue?

**Comment by theirrationalone** [comment]
Please check on which port anvil is running and modify Accordingly.

**Reply by shikhar229169** [reply, marked as answer]
Ensure that anvil chain is running in a terminal.
You can start the anvil chain by typing ```anvil``` in the terminal.

**Reply by DrBailey27** [reply]
Thank you
it worked
